### PR TITLE
Python demo pvspeaker

### PR DESCRIPTION
**Pull request by albho**

### CODE REVIEW

**Comment by bejager** [comment]
Happy to have this merged, waiting for your approval @ksyeo1010

**Review by ksyeo1010 — APPROVED** [review]

**Comment by albho** [comment]
@ksyeo1010 could you please make sure to squash and merge instead of just merging :)

**Pull request merged**
